gadget.yaml

```diff
@@ -14,8 +14,23 @@ volumes:
         offset-write: mbr+92
         content:
           - image: pc-core.img
-      - name: EFI System
+      - name: Recovery
+        filesystem: vfat
+        # the ESP partition
         type: EF,C12A7328-F81F-11D2-BA4B-00A0C93EC93B
+        # max volume label length: 11 chars
+        filesystem-label: sys-recover
+        # too big? too small?
+        size: 600M
```

Comment:
There's a recommended size for windows systems that we should probably follow. I think it's around a GB now.

Reply:
Yeah, Samuele also commented on this. I increased the size for now.